gridformat/vtk/xml.hpp

```diff
@@ -284,14 +284,16 @@ class XMLWriterBase
     }
 
     void _add_meta_data_fields(WriteContext& context) const {
+        const auto& names = this->_meta_data_field_names();
+        if (std::ranges::empty(names))
+            return;
+
         XMLElement& field_data = context.xml_representation
-                                    .get_child(context.vtk_grid_type)
-                                    .get_child("FieldData");
+            .get_child(context.vtk_grid_type).add_child("FieldData");
```

Comment:
you dont like the original indentation and split on two lines?

Feel free to ignore, I'll approve the changes now.

Reply:
Sorry, I‘ll remove the white space change so it‘s easier to see the actual change